FBSnapshotTestCase/Categories/UIImage+Compare.m

```diff
@@ -129,4 +115,64 @@ - (BOOL)fb_compareWithImage:(UIImage *)image tolerance:(CGFloat)tolerance
     return imageEqual;
 }
 
+- (BOOL)comparePixelWithPixelTolerance:(CGFloat)pixelTolerance
```

Comment:
If this is an internal method prefix it with an underscore

Reply:
Done 👍